src/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Composer;
+
+use Composer\Script\Event;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class ScriptHandler
+{
+    protected static $destination = '/src/Bridge/Symfony/Bundle/Resources/public/swagger-ui/dist';
```

Comment:
@dunglas Waiting on an answer to this one, I'm not sure what should be done. Please give me strict instructions and it'll be done by the end of the week ;).